src/Http/Role/AddLabelToRoleRequestHandler.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Role;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Http\ApiProblem\ApiProblem;
+use CultuurNet\UDB3\Http\Request\RouteParameters;
+use CultuurNet\UDB3\Label\ReadModels\JSON\Repository\ReadRepositoryInterface;
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
+use CultuurNet\UDB3\Role\Commands\AddLabel;
+use Fig\Http\Message\StatusCodeInterface;
+use InvalidArgumentException;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Slim\Psr7\Response;
+
+class AddLabelToRoleRequestHandler implements RequestHandlerInterface
+{
+    private CommandBus $commandBus;
+
+    private ReadRepositoryInterface $labelRepository;
+
+    public function __construct(CommandBus $commandBus, ReadRepositoryInterface $labelRepository)
+    {
+        $this->commandBus = $commandBus;
+        $this->labelRepository = $labelRepository;
+    }
+
+    public function handle(ServerRequestInterface $request): ResponseInterface
+    {
+        $routeParameters = new RouteParameters($request);
+        $roleId = $routeParameters->getRoleId();
+
+        try {
+            $roleId = new UUID($roleId);
+        } catch (InvalidArgumentException $e) {
+            throw ApiProblem::roleNotFound($roleId);
+        }
+
+        $labelIdentifier = $routeParameters->get('labelIdentifier');
```

Comment:
Maybe `labelId` is a better parameter name as it matches the naming convention of other parameters better.

Reply:
I'll change that in part 3, as it changes the routing there I can do it in one go.

Reply:
Is not either a label id of a label name? I think that's why it was called "identifier" (although it remains vague)

Reply:
Note how it does `$entity = $this->labelRepository->getByName($labelIdentifier);` in the catch if it's not a valid UUID

Reply:
`labelIdOrName` is maybe the clearest?

Reply:
Cleverly solved @steftrenson !